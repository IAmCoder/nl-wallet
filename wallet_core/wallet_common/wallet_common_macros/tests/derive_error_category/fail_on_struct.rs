#[derive(wallet_common_macros::ErrorCategory)]
struct Error {}

fn main() {}
