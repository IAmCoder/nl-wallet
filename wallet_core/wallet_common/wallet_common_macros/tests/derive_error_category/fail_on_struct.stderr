error: ErrorCategory can only be derived for Enums.
 --> tests/derive_error_category/fail_on_struct.rs:2:1
  |
2 | struct Error {}
  | ^^^^^^
